pyproject.toml

```diff
@@ -65,3 +65,16 @@ doctest_rst = true
 text_file_format = 'rst'
 addopts = '--open-files'
 
+[tool.ruff]
+line-length = 110
+select = ['F', 'W', 'E', 'C']
+ignore = [
+    'C901', # variable is too complex
+]
+exclude = [
+    'docs',
+    'build',
+    'dist',
+    '.tox',
+    '.eggs',
+]
```

Comment:
Looks like a minor formatting issue here?

Reply:
`toml` allows a trailing comma, but I can remove it if you prefer. Is that what you were referencing?

Reply:
No, referring to the popup regarding no newline *after* line 80.

Reply:
oh, I didn't even notice that 😅